projects/meta_cl/networks/dendritic_networks.py

```diff
@@ -107,3 +109,107 @@ def forward(self, x):
         pred = self.prediction(x)
         out = self.gating_layer(pred, context=mod)
         return out
+
+
+class DendriticNetwork2(nn.Module):
```

Comment:
Maybe give this a better name, since we will probably go through many iterations of network architecture. Although, there's an argument to be made for this naming style too (e.g., iPhone3, iPhone4, iPhone5, etc). Do you think we can make it more specific?

Reply:
Fair. We can call it `ANMLDendriticNetwork`, since it's mimicking theirs.

Reply:
I also changed `metacl_dendrites2` to `metacl_anml_dendrites`.